### ISSUE TITLE
Firefox open problem with `Uncaught DOMException: The operation is insecure. sdk-all-min.js:1256`

### ISSUE THREAD

**Issue by ischiros** [ONLYOFFICE/DocumentServer#1549]
Hello,
For a few days now, it's been impossible for me to open .docx .xlsx .odt and .ods documents with Firefox.
I have an error message saying "Une erreur s’est produite lors de l’ouverture du fichier" (aka "An error occured during the file opening") :
<img width="407" alt="Capture d’écran 2021-11-23 à 10 34 20" src="https://user-images.githubusercontent.com/1273505/143000639-2c695b8f-abad-4ce4-9bf4-0146e96cf964.png">

I use Nextcloud 22.1.1, OnlyOffice 7.2.0 and Firefox 94.0.1
With another browser (Safari or Chrome), there is no error.

**Comment by ShockwaveNN** [comment]
Ok, waiting for theirs reply

I've moved this issue to DocumentServer repo, since seems it has nothing to do with nextcloud

**Comment by ShockwaveNN** [comment]
And to be clear - is this issue reproduces on Firefox in incognito mode?

**Comment by ischiros** [comment]
> And to be clear - is this issue reproduces on Firefox in incognito mode?

Yes, it is reproduced.

**Comment by ShockwaveNN** [comment]
I think it may be related to some incorrect configs or broken font install process (which done on backend) so I'll wait until your admins contact us

**Comment by ischiros** [comment]
Hello,

Solution found !
The problem was caused by the third-party cookies, blocked by Firefox.
To solve it, I disabled Firefox advanced protection for our Nextcloud server. Cf attached.
![Capture d’écran 2022-01-06 à 18 42 08](https://user-images.githubusercontent.com/1273505/148428350-4d8afcb1-2008-4164-9b8a-c0517186c4ea.png)

I guess something changed in Onlyoffice behaviour, because I've had these settings in Firefox for years now.
